.bazelrc

```diff
@@ -0,0 +1,13 @@
+build --enable_platform_specific_config
+
+# COPTS
+build:linux --cxxopt=-std=c++17
+build:macos --cxxopt=-std=c++17
+# TODO(mathetake): Windows build is not verified yet.
+# build:windows --cxxopt="/std:c++17"
+
+# LINKOPTS
+# See https://bytecodealliance.github.io/wasmtime/c-api/
+build:linux --action_env=BAZEL_LINKOPTS=-lm:-lpthread:-ldl
+# TODO(mathetake): Windows build is not verified yet.
+# build:windows --action_env=BAZEL_LINKOPTS=ws2_32.lib:advapi32.lib:userenv.lib:ntdll.lib:shell32.lib:ole32.lib
```

Comment:
Please group those by config name, not a feature (i.e. group all `build:linux` together, etc.).